test/NuGet.Clients.Tests/NuGet.VisualStudio.Common.Test/Telemetry/TelemetryUtilityTests.cs

```diff
@@ -204,9 +206,13 @@ public void ToJsonArrayOfTimingsInSeconds_WithOneValue_ReturnsTimingsInSeconds()
             TelemetryUtility.ToJsonArrayOfTimingsInSeconds(values).Should().Be("[5]");
         }
 
-        [Fact]
-        public void ToJsonArrayOfTimingsInSeconds_WithMultipleValues_AppendsValuesWithComma()
+        [Theory]
+        [InlineData("en")]
+        [InlineData("de")]
+        public void ToJsonArrayOfTimingsInSeconds_WithMultipleValues_AppendsValuesWithComma(string culture)
         {
+            Thread.CurrentThread.CurrentCulture = new CultureInfo(culture);
```

Comment:
Is each test run on a different thread?

Reply:
I am not sure about that but have observed that `Thread.CurrentThread.CurrentCulture` resets back to system culture for each test being run.

Reply:
Is that something that's documented? I'm just worried that this has the potential to cause some random test failures in multithreaded tests.

Reply:
Fair point. I couldn't find anything specific in the documentation.
I have, however, stumbled upon [UseCulture Attribute](https://github.com/xunit/samples.xunit/blob/main/UseCulture/UseCultureAttribute.cs) in xunit.samples repository. Maybe that's a way to go for all such tests requiring specific culture being set?